css/styled.css

```
@@ -435,8 +438,198 @@ body {
   padding-top: 25px;
   padding-left: 18px;
 }
+/* END information section */
 
 
+/* START featured section */
+
+.featured {
+  padding-top: 50px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+.featured__title {
+  color: #363535;
+  font-size: 24px;
+  font-weight: 700;
+  margin-bottom: 30px;
+}
+
+.featured__container {
+  display: flex;
+  gap: 20px;
+}
+
+.featured__products-info {
+  width: 60%;
+  background-color: var(--color-white);
+  box-shadow: 0px 0px 4px 0px rgba(54, 53, 53, 0.05);
+  border-radius: 12px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
+}
+
+.fila--1 {
+  border-bottom: solid 1px #a6a6a660;
+  padding-bottom: 10px;
+}
+
+.col--1 {
+  border-right: solid 1px #a6a6a660;
+  padding-bottom: 10px;
+}
+
+.featured__products-card {
```

A esta le pondria `featured__product`
```suggestion
.featured__product {
```